clang/include/clang/Basic/DiagnosticParseKinds.td

```diff
@@ -868,6 +868,9 @@ def err_requires_expr_in_simple_requirement : Error<
   "requires expression in requirement body; did "
   "you intend to place it in a nested requirement? (add another 'requires' "
   "before the expression)">;
+def err_missing_template_arg_list_after_template_kw : Error<
+  "a template argument list is expected after a name prefixed by the template "
+  "keyword">;
```

Comment:
This should be a pedantic warning (maybe defaulting to an error)
Otherwise we might break existing (non conforming) code

https://github.com/search?q=%2Fs%5C.template+%5Ba-zA_Z_%5D%2B%5B%28%5D%2F+language%3AC%2B%2B&type=code

Reply:
> This should be a pedantic warning (maybe defaulting to an error) Otherwise we might break existing (non conforming) code

I've done this by adding this as a warning to the Pedantic diagnostic group, but I've had to increment the number in `test/Misc/warning-flags.c:91`, and there's a rather stern warning about not adding to `-Wpedantic`. Would this be ok?